device-types/Infoblox/IB-825.yaml

```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Infoblox
+model: IB-825
```

Comment:
Is the device actually called `Trinzic 825`?

Reply:
Trinzic would be the original name of the company from long ago before it was Infoblox.  I go by what our NetMRI tool discovers it as (shown below).  NetMRI is a Network Management platform from Infoblox.
![image](https://github.com/netbox-community/devicetype-library/assets/16478194/a45029f7-db99-4e7d-8ce2-56384615dc63)

Reply:
Daniel - I took a look at what our actual product sheet said for these devices and it uses TE-805-HW-AC.  That is the base hardware.  The '2' (in the 825) is addition software options.  Splitting hairs here, but wanted to be fair on what direction you wanted to go with this.

![image](https://github.com/netbox-community/devicetype-library/assets/16478194/b3e909f6-856f-44c8-be3c-20925e605efd)

Reply:
So the hardware is what we are modelling here. I think it is best to make the part number equal what the hardware is and then have the model number reflect IB-825

Reply:
I would agree to that.  I'll try and update the others with the part number as well next week.